pqarrow/arrow.go

```diff
@@ -146,7 +268,7 @@ func contiguousParquetRowGroupToArrowRecord(
 			default:
 				name := field.Name()
 				if distinctColumns[0].Match(name) {
-					typ, nullable, array, err := parquetColumnToArrowArray(
+					_, _, array, err := parquetColumnToArrowArray(
```

Comment:
now that we don't need the type and nullable property anymore we should probably separate those parts out of that function